base/loading.jl

```diff
@@ -3288,7 +3288,7 @@ function compilecache(pkg::PkgId, path::String, internal_stderr::IO = stderr, in
     if p.exitcode == 125
         return PrecompilableError()
     else
-        error("Failed to precompile $(repr("text/plain", pkg)) to $(repr(tmppath)).")
```

Comment:
This seems like it could be a bit confusing?
```
julia> p
Process(`./julia -e 'ccall(:jl_raise, Cvoid, (Cint,), 2)'`, ProcessSignaled(2))

julia> p.exitcode
0

julia> Base.process_status(p)
"ProcessSignaled(2)"
```

Reply:
Ack. So what I want is to embed `$(Base.process_status(p))` instead of just `p.exitcode`?